scopus-import/src/main/java/no/sikt/nva/scopus/conversion/CristinConnection.java

```diff
@@ -45,31 +40,21 @@ public Optional<Person> getCristinPersonByCristinId(URI cristinPersonId) {
                                        .map(this::getCristinResponse)
                                        .map(this::getBodyFromResponse)
                                        .map(this::getCristinPersonResponse)
-                                       .orElse(this::logFailureForPersonAndReturnNull));
+                                       .orElse(null));
     }
 
     public Organization getCristinOrganizationByCristinId(URI cristinOrgId) {
         return attempt(() -> createRequest(cristinOrgId))
                    .map(this::getCristinResponse)
                    .map(this::getBodyFromResponse)
                    .map(this::convertToOrganization)
-                   .orElse(this::logFailureFetchingOrganizationAndReturnNull);
+                   .orElse(null);
```

Comment:
Tenker det er unødvendig å logge "could not extract cristin organization", vi får jo 200 i response fra cristin, det er bare at organisasjon ikke er i cristin/cristinOrgId fra pia er null og det er greit. Dersom vi får 400 i repsonse => blir det logget